.eslintrc.js

```diff
@@ -1,108 +1,29 @@
-// The ESLint browser environment defines all browser globals as valid,
-// even though most people don't know some of them exist (e.g. `name` or `status`).
-// This is dangerous as it hides accidentally undefined variables.
-// We blacklist the globals that we deem potentially confusing.
-// To use them, explicitly reference them, e.g. `window.name` or `window.status`.
-// See https://github.com/facebook/create-react-app/blob/3c2f2d4/packages/eslint-config-react-app/index.js#L19-L24
-const restrictedGlobals = require('confusing-browser-globals');
+'use strict';
 
+// eslint-disable-next-line  no-unused-vars
```

Comment:
What's up with these vars anyway? Can we add a comment so to why they are here?

Reply:
added a comment